tools/src/dbgen/Lib.hs

```diff
@@ -265,6 +277,25 @@ generateFakeTxs SimpleTxsHistory{..} aId   = do
     -- Insert into the @WalletStorage@.
     insertIntoHistoryCache db walletId fakeMapTxs
   where
+    -- | Generate sensible @Tx@.
+    genTxs :: Gen Tx
+    genTxs = do
+
+        -- Generate a more realistic distribution. @Martoon said:
+        -- After release we have limit - ~76 inputs in transaction,
+        -- and bittrex worked for a month or two before hitting that bound.
+        -- In other words, it should be pretty rare to see that stuff.
```

Comment:
Umm, it's not necessary to refer to my words )
Also I have found that mentions of any concrete partners are strongly discouraged.
Just say that it was an approximate limit one day.

Reply:
Well, I had the best intention. It seemed kind of wrong to literally copy your words and not give you any credit.
I thought about "concrete partners" as well, it should probably be visible on explorer anyway, but ok.

Reply:
> It seemed kind of wrong to literally copy your words and not give you any credit.
To allow people blame me later :peka: :smile: